api/v1/services/RegistrationService.js

```diff
@@ -222,18 +222,15 @@ function _addToMailingList(attendee, decision) {
 }
 
 /**
- * Determines whether or not an attendee has at least one
- * project or ecosystem interest
- * @param  {Array}  projects	the projects list (or undefined)
- * @param  {Array}  ecosystemInterests the ecosystem interests list (or undefined)
+ * Returns true, since proejcts and ecosystems have been removed
  * @return {Boolean} whether or not the pairing is valid
  */
-function _hasValidAttendeeAssignment(projects, ecosystemInterests) {
-  return (!!projects && projects.length > 0) || (!!ecosystemInterests && ecosystemInterests.length > 0);
+function _hasValidAttendeeAssignment() {
```

Comment:
I'd prefer if we didn't gut the method like this. You need to determine if these checks are replaced by new fields that you've added, and if not change the logic of any code that used to call this function.

Reply:
Function now check for os contributor 6a2e2c84e513e9b0f05760bc0318d111b439fc34